src/app/core/learning-object-module/hierarchy/hierarchy.service.ts

```diff
@@ -60,12 +59,11 @@ export class HierarchyService {
   /**
    * Adds children to a learning object
    *
-   * @param username the username of the author
    * @param object the object having children
    * @param children the children to be had
    * @returns
    */
-  async addChildren(username: string, object: any, children): Promise<any> {
+  async addChildren(object: any, children): Promise<any> {
```

Comment:
What type is object? Is it LearningObject?

Reply:
Yes, the types for both parameters is LearningObject